static-site/src/components/Showcase/index.tsx

```diff
@@ -1,25 +1,22 @@
 /* eslint-disable react/prop-types */
-import React, { useCallback, useEffect, useState } from 'react';
+import React, { useState } from 'react';
 import { FaChevronDown, FaChevronUp } from 'react-icons/fa';
 import styled from 'styled-components';
-import {CardInner, CardImg, CardTitle} from "../Cards/styles";
+import { CardImg } from "../Cards/styles";
 import { theme } from "../../layouts/theme";
-import { goToAnchor } from '../../../vendored/react-scrollable-anchor/index';
-import { createFilterOption } from "./useFilterOptions";
-import { LIST_ANCHOR } from "./index";
-import { Card, FilterOption, Group } from './types';
+import { Card } from './types';
 
 const cardWidthHeight = 160; // pixels
 const expandPreviewHeight = 50 //pixels
 const transitionDuration = "0.3s"
 const transitionTimingFunction = "ease"
 
-interface ShowcaseProps {
-    cards: Card[]
-    setSelectedFilterOptions: React.Dispatch<React.SetStateAction<readonly FilterOption[]>>
+interface ShowcaseProps<AnyCard extends Card> {
+    cards: AnyCard[]
+    CardComponent: React.FunctionComponent<{ card: AnyCard }>
 }
 
-export const Showcase = ({cards, setSelectedFilterOptions}: ShowcaseProps) => {
+export const Showcase = <AnyCard extends Card>({cards, CardComponent}: ShowcaseProps<AnyCard>) => {
```

Comment:
[using this line for the comment, but I haven't dug into where this behaviour is originating from]

[not blocking]

The `<ArrowButton>` is rendering as quite different sizes within the splash page vs /pathogens, with the former too small!
<img width="1175" alt="image" src="https://github.com/nextstrain/nextstrain.org/assets/8350992/7908a22d-4f00-4e41-9abb-d9c9eea9eeff">

Reply:
Oh, It's because `<ListResources>` is wrapped in `<ListResourcesContainer>` which sets `font-size: 1.8rem;`. Suggest changing the `<ArrowButton>` to itself explicitly define `font-size: 1.8rem;`

Reply:
Good eye! Fixed: d038264bf9abe040eebca7486b4b528f6f0f3387